@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
All matched files use Prettier code style!
 ✓ scripts/tests/prettierFormatting.test.ts (1 test) 12110ms
   ✓ Prettier formatting > frontend sources are formatted  12109ms
 ✓ tests/eslintPluginPresence.test.ts (1 test) 2ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 2ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 2ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 2ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 2ms
 ✓ tests/prismSvelte.test.ts (1 test) 8ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 2ms
 ✓ tests/progressionBalance.test.ts (1 test) 16ms
 ✓ tests/domException.test.ts (1 test) 2ms
 ✓ frontend/tests/sanity.test.ts (1 test) 2ms
 ✓ tests/indexeddb.server.test.ts (1 test) 111ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  frontend/__tests__/Quests.test.js > Quests Component > falls back to safe internal quest routes for custom quests with unsafe hrefs
AssertionError: expected undefined to be '/quests/custom/unsafe' // Object.is equality

- Expected: 
"/quests/custom/unsafe"

+ Received: 
undefined

 ❯ frontend/__tests__/Quests.test.js:192:59
    190| 

    191|             const unsafeQuestLink = host.querySelector("a[data-questid…
    192|             expect(unsafeQuestLink?.getAttribute('href')).toBe('/quest…
       |                                                           ^
    193| 
    194|             const externalQuestLink = host.querySelector("a[data-quest…

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

 Test Files  1 failed | 290 passed (291)
      Tests  1 failed | 1023 passed (1024)
   Start at  05:38:28
   Duration  204.99s (transform 5.03s, setup 4.24s, collect 20.17s, tests 54.60s, environment 85.74s, prepare 14.34s)


Error: AssertionError: expected undefined to be '/quests/custom/unsafe' // Object.is equality

- Expected: 
"/quests/custom/unsafe"

+ Received: 
undefined

 ❯ frontend/__tests__/Quests.test.js:192:59


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/__tests__/Quests.test.js
+++ b/frontend/__tests__/Quests.test.js
@@ -187,6 +187,9 @@ describe('Quests Component', () => {
             mountedComponent = mount(Quests, { target: host, props: { quests } });
             await vi.runAllTimersAsync();
             await vi.waitFor(() => expect(listCustomQuests).toHaveBeenCalled());
+            await vi.waitFor(() =>
+                expect(host.querySelector("a[data-questid='custom/unsafe']")).not.toBeNull()
+            );
 
             const unsafeQuestLink = host.querySelector("a[data-questid='custom/unsafe']");
             expect(unsafeQuestLink?.getAttribute('href')).toBe('/quests/custom/unsafe');

